Readme.md

```diff
@@ -320,6 +320,8 @@ test('call the callback', () => {
   fn(callback)
   expect(callback).toBeCalled()
   expect(callback.mock.calls[0][1].baz).toBe('pizza') // Second argument of the first call
+  // Use a matcher for the second parameter:
```

Comment:
Maybe something like this would be more clear:

```suggestion
  // Match the first and the last arguments but ignore the second argument
```

Reply:
Definitely!